dotCMS/src/integration-test/java/com/dotmarketing/startup/runonce/Task05370ChangeContainerPathToAbsoluteTest.java

```diff
@@ -0,0 +1,355 @@
+package com.dotmarketing.startup.runonce;
+
+import com.dotcms.datagen.SiteDataGen;
+import com.dotcms.datagen.TemplateDataGen;
+import com.dotcms.datagen.ThemeDataGen;
+import com.dotcms.rendering.velocity.viewtools.DotTemplateTool;
+import com.dotcms.util.IntegrationTestInitService;
+import com.dotmarketing.beans.Host;
+import com.dotmarketing.business.CacheLocator;
+import com.dotmarketing.common.db.DotConnect;
+import com.dotmarketing.exception.DotDataException;
+import com.dotmarketing.exception.DotSecurityException;
+import com.dotmarketing.portlets.contentlet.model.Contentlet;
+import com.dotmarketing.portlets.templates.model.Template;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.HashMap;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertTrue;
+
+public class Task05370ChangeContainerPathToAbsoluteTest {
+    final String body =
+        "<html>" +
+            "<head>" +
+                "#dotParse('%1$s/application/themes/landing-page/html_head.vtl')" +
+                "<link rel=\"stylesheet\" type=\"text/css\" href=\"/html/css/template/reset-fonts-grids.css\" />" +
+            "</head>" +
+            "<body>" +
+                "<div id=\"resp-template\" name=\"globalContainer\">" +
+                    "<div id=\"hd-template\">" +
+                    "#dotParse('%1$s/application/themes/landing-page/header.vtl')" +
+                "</div>" +
```

Comment:
Codacy found an issue: [The String literal "</div>" appears 11 times in this file; the first occurrence is on line 36](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5990100)